### ISSUE TITLE
Custom audio export. libopus is 'incompatible' codec for opus container

### ISSUE THREAD

**Issue by Penikov** [audacity/audacity#2117]
**Describe the bug**
libopus is 'incompatible' codec for opus container in the custom expor dialog.

**To Reproduce**
1. Go to custom export
2. choose opus container in the list of containers
3. choose libopus codec in the list of codecs
4. Try to export audio

**Additional information (please complete the following information):**
 - OS: Windows 10
 - Audacity 3 +

**Comment by SteveDaulton** [comment]
Confirmed on Linux.
Also  reported here (Windows) https://forum.audacityteam.org/viewtopic.php?f=46&t=125605

**Comment by dozzzzer** [comment]
Still reproducible as of 3.4.2

<img width="450" alt="image" src="https://github.com/audacity/audacity/assets/70063970/e6fdb52a-4da5-47bd-bfd1-3dba2afc214b">

**Comment by crsib** [comment]
Yeah, this bug wouldn't fix itself. Instead of querying codecs and supported containers, Audacity has a limited hardcoded list.
